parsec/backend/postgresql/user_queries/find.py

```diff
@@ -88,6 +108,32 @@ async def query_find(
     return results, len(all_results)
 
 
+@query()
+async def query_retrieve_human(
+    conn, organization_id: OrganizationID, arguments: List[Tuple[str, str, str]]
+) -> HumanFindResultItem:
+    if len(arguments) == 0:
+        return
+
+    kwargs = {"organization_id": organization_id, "now": pendulum_now()}
+    factory_arguments = tuple()
+    for (field, operator, value) in arguments:
+        key = field.replace(".", "_")
+        factory_arguments += (field, operator, f"${key}")
+        kwargs[key] = value
+    else:
```

Comment:
why the `else` here (given there is no `break` in the for, we always go into the `else`)